faq.rst

```diff
@@ -112,6 +112,46 @@ explicitly with ::
 
 https://en.wikipedia.org/wiki/EGL_(API)
 
+How to achieve transparency with 2D objects?
+--------------------------------------------
+
+It is possible to render 2D visuals, such as ``ImageVisual``, with translucency
+(i.e. partial transparency, a see-through effect). Here are the key points to
+achieve this, using ``ImageVisual`` as an example:
+
+1. Set the opcity value in the alpha channel of the images: ::
+  
+    # A white image with integer values between 0 and 255.
+    image_data1 = np.ones((200, 300, 4), dtype='uint8')
+    # Half translucent.
+    image_data1[..., 3] = 128
+    # A blue image with float values between 0 and 1.
+    image_data2 = np.zeros((200, 300, 4), dtype='np.float32')
+    image_data2[..., 2]  = 1.0  # Blue.
+    # A bit more translucent.
+    image_data2[..., 3] = 0.25
+    visual1 = ImageVisual(image_data1)
+    visual2 = ImageVisual(image_data2)
+
+2. Enable translucency for the visuals: ::
+
+    # This is the default for ImageVisual, so it could be skipped, but it could
+    # be needed with other types of visuals.
+    visual1.set_gl_state('translucent')
+    visual2.set_gl_state('translucent')
+
+3. Position the visuals at different depth levels (z-levels): ::
+
+    visual1.transform = vispy.STTransform(translate=(0, 0, 1)
+    visual2.transform = vispy.STTransform(translate=(0, 0, 2)
+
+   A higher ``z`` value means further back.
```

Comment:
This depends on the camera for a SceneCanvas and the properties of that camera. Maybe add a "by default" or something.